SimTracker/SiStripDigitizer/plugins/SiLinearChargeDivider.h

```diff
@@ -13,6 +13,9 @@
 
 #include "SimGeneral/HepPDTRecord/interface/ParticleDataTable.h"
 
+#include <fstream>
+#include <string>
```

Comment:
Please move these includes to the `.cc` as there is nothing in the header requiring them.